docs/types.rst

```diff
@@ -857,13 +859,18 @@ or create a new memory array and copy every element.
         }
     }
 
-.. index:: ! array;literals, !inline;arrays
+.. index:: ! array;literals, ! inline;arrays
+
+Array Literals
+^^^^^^^^^^^^^^
 
-Array Literals / Inline Arrays
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+An array literal is a comma-separated list of one or more expressions, enclosed in square
+brackets (``[...]``). For example ``[1, a, f(3)]``. The elements must have a
+type so they can all be implicitly converted to and this is the elementary type of the array.
+Array literals are always statically-sized memory arrays.
 
-Array literals are arrays that are written as an expression and are not
-assigned to a variable right away.
+In the example below, the type of ``[1, 2, 3]`` is
+``uint8[3] memory``. Because the type of each of these constants is ``uint8``, if you want the result to be a ``uint[3] memory`` type, you need to convert the first element to ``uint``.
```

Comment:
Can you start a sentence with `because`?

Reply:
You can in certain cases, and this one is OK :)